ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/ControlLabels.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Label, LabelGroup } from '@patternfly/react-core';
+
+import { ComplianceControl } from 'services/ComplianceCommon';
+
+type ControlLabelsProps = {
+    controls: ComplianceControl[];
+    numLabels?: number;
+};
+
+function ControlLabels({ controls, numLabels = Infinity }: ControlLabelsProps): React.ReactElement {
+    return (
+        <LabelGroup numLabels={numLabels}>
+            {controls.map(({ control, standard }) => (
+                <Label key={standard}>{`${standard} ${control}`}</Label>
```

Comment:
> Warning: Each child in a list should have a unique "key" prop.

The first negative I have seen with `TbodyUnified` that it takes credit for the problem.

A change to `key={control}` removes the warning in the particular case, but can we assume only one standard? If no, then `key` prop might need same value as rendered text.

Reply:
interesting, yeah there is no indicator as to where that issue is actually occurring. Luckily `control` here should be unique although we could always just concatenate `standard` and `control`.